### ISSUE TITLE
Skipped Heading Hierarchy on Dashboard

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#1961]
### Descriptive summary

In Hyrax 2.0.0 rc1 on the dashboard, the heading hierarchy is out of order and skips heading level.

### Rationale

Headings provide document structure and facilitate keyboard navigation by users of assistive technology. These users may be confused or experience difficulty navigating when heading levels are skipped. Heading levels should be selected based on content not visual weight. 

### Expected behavior

Expected properly nested `<h1>` through `<h6>` without skipping levels.

### Actual behavior

Modal is at the top of the document and starts with an `<h2>` and the content jumps from `<h1>` to an `<h3>`.

![screen shot 2017-10-25 at 2 02 27 pm](https://user-images.githubusercontent.com/4163828/32014859-4c0a1032-b98d-11e7-8f34-9977ddbd5585.png)

### Steps to reproduce the behavior

1. Use an accessibility validator or control click to inspect page
1. View heading hierarchy

**Comment by adamjarling** [comment]
Here's an updated WAVE output of the Hyrax Dashboard September 2018:

![dashboard-wave-output](https://user-images.githubusercontent.com/3020266/45172550-45a94a00-b1cb-11e8-8251-f560f20d0338.png)

Looks like the "Done Looks Like" for this ticket should be updated to:

- [ ] item #1....

**Comment by adamjarling** [comment]
This is fixed, just waiting on the build to get cleaned up before pushing for a PR.
